addons/web/static/src/views/form/form_controller.scss

```diff
@@ -237,10 +237,17 @@
 
         padding-top: map-get($spacers, 2);
         padding-left: var(--formView-sheetBg-padding-x);
-        padding-right: var(--formView-sheetBg-padding-x);
+        padding-right: var(--formView-sheetBg-padding-right, var(--formView-sheetBg-padding-x));
         width: 100%;
         max-width: $o-form-view-sheet-max-width;
         margin-right: auto; // Always align to le left
+
+        // Reduce horizontal spacing if the next sibling is an aside chatter
+        &:has( + .oe_chatter.o-aside) {
+            @media (max-width: 1920px) {
+                --formView-sheetBg-padding-right: #{map-get($spacers, 2)};
+            }
+        }
```

Comment:
`oe_chatter` is a `mail` concern and not `web`.

Would be best to move this CSS in `mail` file, e.g. `form_renderer.scss`.